src/features/movies/MoviePage/moviePageSlice.js

```diff
@@ -0,0 +1,44 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const moviePageSlice = createSlice({
+  name: "moviePage",
+  initialState: {
+    status: "loading",
+    cast: [],
+    crew: [],
+  },
+  reducers: {
+    fetchMoviePage: () => ({
```

Comment:
tak jak przy personPage, myślę, że można tutaj spokojnie nazwać po prostu fetchMovie :), a przy pobieraniu cast/crew - fetchMovieDetails ;) nazwę samego slice'a też w sumie można uprościć :) ale to tylko moja sugestia :)